src/lightning/fabric/plugins/precision/bitsandbytes.py

```diff
@@ -164,11 +168,36 @@ def _quantize_on_load_hook(quantize_fn: Callable[[torch.Tensor], None], state_di
 
 
 def _ignore_missing_weights_hook(module: torch.nn.Module, incompatible_keys: _IncompatibleKeys) -> None:
+    # since we manually loaded the weight in the `_quantize_on_load_hook` hook, we need to avoid this missing key false
+    # positive
     for key in reversed(incompatible_keys.missing_keys):
         if key.endswith("weight"):
             incompatible_keys.missing_keys.remove(key)
 
 
+def _replace_param(
+    p: torch.nn.Parameter, data: torch.Tensor, quant_state: Optional[Tuple] = None
```

Comment:
```suggestion
    param: torch.nn.Parameter, data: torch.Tensor, quant_state: Optional[Tuple] = None
```
lets pls use more then one letter